src/Components/App/App.css

```diff
@@ -1,9 +1,37 @@
 :root {
-  --primary-bkgd-color: rgb(255, 204, 0);
-  --secondary-bkgd-color: rgb(254, 84, 84);
-  --tertiary-bkgd-color: rgb(230, 229, 229);
-  --hover-bkgd-color: ivory;
-  --bkgd-radial-gradient: content-box radial-gradient(var(--secondary-bkgd-color), var(--primary-bkgd-color));
+  --element-bkgd-color: rgba(255,255,255, .4);
+  --border-radius: 2px;
+  --font-family: 'Plus Jakarta Sans', sans-serif;
+}
+
+.uni-btn {
+  border-radius: var(--border-radius);
+  background-color: none;
+  border: solid white 1px;
+  padding: 1vmin 2vmin 1vmin 2vmin;
+  margin: 1vmin;
+  height: fit-content;
+  cursor: pointer;
+  text-decoration: none;
+  transition: white .5s;
+  font-weight: bold;
```

Comment:
I really like the `transition: background-color .5s` property for this. I think I had it on the feelings originally. Then when you hover (like you have below), there's a nice, smooth transition that feels cohesive with the softness of our app

Reply:
Agree! I fully swiped that from your button design because it was so choice. Appreciate you dude!

Reply:
To clarify, the way it's set up right now doesn't utilize the transition. Instead of `transition: white .5s;`, you would need to have `transition: background-color .5s;` - Then (like you have it), you change the background color in the hover state

Reply:
I don't love the black text with the white button design. I think maybe it should be all black or all white? Just a thought!